ParkingZoneApp/IRepository/IRepository.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ParkingZoneApp.IRepository
+{
+    public interface IRepository<T>
```

Comment:
Create a folder called "Interfaces" in the Repositories folder and add them to that folder